<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-H7F0JOM" failures="0" tests="5" name="Test31" time="0.13" errors="0" timestamp="2022-05-05T14:07:58 EEST">
  <testcase classname="example_tests.ExceptionTest" name="f" time="0.014"/>
  <testcase classname="example_tests.ExceptionTest" name="f2" time="0.002"/>
  <testcase classname="example_tests.ExceptionTest" name="f3" time="0.002"/>
  <testcase classname="example_tests.ExceptionTest" name="f4" time="0.007"/>
  <testcase classname="example_tests.ExceptionTest" name="f5" time="0.004"/>
  <testcase classname="example_tests.ExceptionTest" name="f1" time="0.0">
    <ignored/>
  </testcase> <!-- f1 -->
</testsuite> <!-- Test31 -->
